### PR TITLE
Fixed Price typo

### PR DESCRIPTION
**Pull request by olawanlejoel**
I suspect there was a typo there! I decided to fix, so it doesn't confuse others.

### CODE REVIEW

File: gsod.md [hunk 1]
```diff
@@ -53,8 +53,8 @@ The project itself will take approximately six months to complete, this would be
 | Budget item   | Amount | Running Total | Notes/justifications |
 | ------------------- | ------------------- | --------------| ----------- |
 | Technical writer/s audit, update, test, and publish new Mautic documentation    |  10,500    | 10,500.00 | |
-| Volunteer stipends  |  500      | 12,000 | 3 volunteer stipends x 500 each |
-| Project t-shirts (5 t-shirts)   | 13,000    |1300 | Design and print T-Shirt for participants and volunteers|
+| Volunteer stipends  |  500      | 12,000.00 | 3 volunteer stipends x 500 each |
```
Comment (RCheesley) [inline gsod.md]:
```suggestion
| Volunteer stipends  |  1,500      | 12,000 | 3 volunteer stipends x 500 each |
```

Thanks for picking this up - actually it should be 1500 (3x 500) and the running total therefore 12,000 so I've tweaked that in my suggestion above.

File: gsod.md [hunk 2]
```diff
@@ -53,8 +53,8 @@ The project itself will take approximately six months to complete, this would be
...
+| Project t-shirts (5 t-shirts)   | 1,000    |13,000.00 | Design and print T-Shirt for participants and volunteers|
```
Comment (RCheesley) [inline gsod.md]:
```suggestion
| Project t-shirts (5 t-shirts)   | 200    |12,200 | Design and print T-Shirt for participants and volunteers|
| Administration costs   | 800    |13,000.00 | Administration expenses ($174 for OysterHR contract, $500 toward attending next in-person conference, misc expenses)|

```

@fakela I added the line for misc admin expenses such as our OysterHR contracting fees, supporting the person to attend our next in-person conference, and a small amount left over for any misc expenses such as postage etc. I think it was missing from the original proposal and got rolled into the tshirts line by mistake.

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for jumping on this @olawanlejoel and sorry for the delay in reviewing!

Just a couple of suggestions as I think the numbers are a bit wonky :)

**Comment by olawanlejoel** [comment]
Hi @RCheesley. Thanks so much!

**Review by RCheesley — APPROVED** [review]
LGTM! 

🚀

**Pull request merged**